### ISSUE TITLE
letsencrypt-auto --no-self-upgrade does not bootstrap

### ISSUE THREAD

**Issue by pde** [letsencrypt/letsencrypt#2432]
From a test farm test:
```
[Process-2 : client 0 ami-26d5af4c ubuntu15.10]
server ec2.Instance(id='i-2085f6a0') at 54.173.22.73
ubuntu@54.173.22.73
[ubuntu@54.173.22.73] put: letest-1455160422/le.tar.gz -> /home/ubuntu/le.tar.gz
[ubuntu@54.173.22.73] run: tar xzf le.tar.gz
[ubuntu@54.173.22.73] out: To run a command as administrator (user "root"), use "sudo <command>".
[ubuntu@54.173.22.73] out: See "man sudo_root" for details.
[ubuntu@54.173.22.73] out: 
[ubuntu@54.173.22.73] out: 

[ubuntu@54.173.22.73] put: scripts/test_letsencrypt_auto_certonly_standalone.sh -> /home/ubuntu/test_letsencrypt_auto_certonly_standalone.sh
[ubuntu@54.173.22.73] run: ./test_letsencrypt_auto_certonly_standalone.sh 
[ubuntu@54.173.22.73] out: To run a command as administrator (user "root"), use "sudo <command>".
[ubuntu@54.173.22.73] out: See "man sudo_root" for details.
[ubuntu@54.173.22.73] out: 
[ubuntu@54.173.22.73] out: + cd letsencrypt
[ubuntu@54.173.22.73] out: + ./letsencrypt-auto certonly --no-self-upgrade -v --standalone --debug --text --agree-dev-preview --agree-tos --renew-by-default --redirect --register-unsafely-without-email --domain ec2-54-173-22-73.compute-1.amazonaws.com --server http://172.31.54.176:4000/directory
[ubuntu@54.173.22.73] out: Creating virtual environment...
[ubuntu@54.173.22.73] out: ./letsencrypt-auto: 428: ./letsencrypt-auto: virtualenv: not found
[ubuntu@54.173.22.73] out: 

ami-26d5af4c - ubuntu15.10 FAIL


letsencrypt.log
--------------------------------------------------------------------------------

[ubuntu@54.173.22.73] sudo: if [ -f /var/log/letsencrypt/letsencrypt.log ]; then     cat /var/log/letsencrypt/letsencrypt.log; else echo "[novarlog]"; fi
[ubuntu@54.173.22.73] out: [novarlog]
[ubuntu@54.173.22.73] out: 

[ubuntu@54.173.22.73] sudo: if [ -f ./letsencrypt.log ]; then     cat ./letsencrypt.log; else echo "[nolocallog]"; fi
[ubuntu@54.173.22.73] out: [nolocallog]
[ubuntu@54.173.22.73] out: 
```

**Comment by erikrose** [comment]
Yes, that's true. It was never really intended as a public flag; it just sort of got promoted. It was intended to mean "Okay, I've bootstrapped and self-upgraded, now install the client." I'll see what I can do.

**Comment by bmw** [comment]
Opening this again because #2451 was reverted. If we are happy with that fix, we can simply revert #2492 before the release.
